### DISCUSSION TITLE
1.63 update

### DISCUSSION THREAD

**Discussion by evanrich** [librenms/docker#368] [Q&A]
can we get 1.63 released?  looks like it was made available 11 days ago, but docker hasn't been updated in 25 days.   thanks!  I'm specifically waiting for "Detection, sensors and zpool/dataset usage for FreeNAS, TrueNAS (#11474) - TheMysteriousX"

**Comment by crazy-max** [comment]
@evanrich Sorry I missed it. Will be available asap.

**Comment by crazy-max** [comment]
@evanrich Done
